src/web-workers/svgo.js

```diff
@@ -60,7 +60,7 @@ const pluginsData = {
   inlineStyles,
   minifyStyles,
   convertStyleToAttrs,
-  cleanupIDs,
+  // cleanupIDs,
   //prefixIds,
```

Comment:
🤔 not seeing the reference to this variable. and can we add a quick note why they are commented out?